common.yml

```
@@ -476,365 +476,365 @@ translations:
     - key: options.job_title.cto
       t: CTO
     - key: options.job_title.front_end_developer
-      t: Front End Developer/Engineer
+      t: Desenvolvedor ou Engenheiro Front End
```

O padrão costuma ser com as palavras juntas. E também é o termo que possui mais resultados em buscas no Google, então acho que seria legal alterar:

```suggestion
      t: Desenvolvedor ou Engenheiro Front-End
```